pkg/storage/internalstorage/register.go

```diff
@@ -86,14 +89,28 @@ func newLogger(cfg *Config) (logger.Interface, error) {
 	if cfg.Log.Stdout {
 		logWriter = os.Stdout
 	} else {
-		if err := os.MkdirAll("/var/log/clusterpedia", 0644); err != nil {
-			return nil, fmt.Errorf("mkdir log dir error: %w", err)
+		loggerConf := cfg.Log.Logger
```

Comment:
`loggerConf` doesn't look clear, and it is not a config. `lumberjackLogger` looks better than `loggerConfig`.